modules/ui/app/scripts/services/UtilitiesFactory.js

```diff
@@ -1002,16 +1002,22 @@ angular.module('wasabi.services').factory('UtilitiesFactory', ['Session', '$stat
 
             changeState: function (experiment, state, afterUpdateFunction) {
                 var stateChange = 'start',
-                    that = this;
+                    that = this,
+                    title = 'Confirm State Change';
                 switch (state.toLowerCase()) {
                     case 'paused':
                         stateChange = 'stop';
                         break;
                     case 'terminated':
                         stateChange = 'terminate';
+                        title = 'Permanently Terminate Experiment';
                         break;
                 }
-                DialogsFactory.confirmDialog('Are you sure you want to ' + stateChange + ' the experiment ' + experiment.label + '?', 'Confirm State Change',
+                var msg = 'Are you sure you want to ' + stateChange + ' the experiment ' + experiment.label + '?';
+                if (state.toLowerCase() === 'terminated') {
+                    msg = 'Are you sure you want to <span style="font-weight: bold;">PERMANENTLY TERMINATE</span> the experiment ' + experiment.label + '?';
```

Comment:
this double confirmation is a good idea 👍 
Will there be a help icon explaining a bit more about permanently terminate state...?